ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/dot1x.schema.yml

```diff
@@ -44,3 +44,93 @@ keys:
             max: 9236
             convert_types:
               - str
+      aaa:
+        type: dict
+        description: Configure AAA parameters.
+        keys:
+          unresponsive:
+            type: dict
+            description: Configure AAA timeout options.
+            keys:
+              eap_response:
+                type: str
+                description: EAP response to send.
+                valid_values:
+                  - "success"
+                  - "disabled"
+              action:
+                type: dict
+                description: Set action for supplicant when AAA times out.
+                keys:
+                  apply_cached_results:
+                    type: bool
+                    description: Use results from a previous AAA response.
+                  cached_results_timeout:
+                    type: dict
+                    keys:
+                      time_duration:
+                        type: int
+                        convert_types:
+                          - str
+                        min: 1
+                        description: |-
+                          Enable caching for a specific duration -
+                          <1-10000>      duration in days
+                          <1-14400000>   duration in minutes
+                          <1-240000>     duration in hours
+                          <1-864000000>  duration in seconds
+                      time_duration_unit:
+                        type: str
+                        required: true
+                        valid_values:
+                          - "days"
+                          - "hours"
+                          - "minutes"
+                          - "seconds"
+                  apply_alternate:
+                    type: bool
+                    description: Apply alternate action if primary action fails.
+                  traffic_allow:
+                    type: bool
+                    description: Set action for supplicant traffic when AAA times out.
+                  traffic_allow_vlan:
```

Comment:
Do we need to mention the allow vlan range and maybe error when out of range?
```
coi414(config-dot1x)#aaa unresponsive action traffic allow vlan ?
  <1-4094>  Identifier for a Virtual LAN
  ```
  Right now I can set any int here and it would render the configs
  ```
    aaa:
    unresponsive:
      eap_response: disabled
      action:
        traffic_allow_vlan: 5000
 ```
 renders this:
 `   aaa unresponsive action traffic allow vlan 5000`

Reply:
Added the min/max range for VLAN, it will throw error when out of range.